README.md

```diff
@@ -1,40 +1,133 @@
+    `UDW` | An object containing the necessary credential information for connecting to the Unizin Data Warehouse, where data will be pulled from.
+
 
 ### Installation & Usage
 
-#### With Docker
 
-You can run the application without installing the dependencies manually by leveraging the `Dockerfile` and Docker 
-Desktop. To run with Docker, do the following:
+#### Docker is coming!
+
+A `docker-compose.yml` file -- that will let you run and test the MySQL database population -- is in the works! For now, please use `virtualenv` (see below).
 
-1. Build an image.
-   ```
-   docker build -t inventory .
-   ```
-1. Run a container using the tagged image, specifying a bind mount to make the output CSVs accessible.
-  ```
-  docker run --mount type=bind,source={some absolute path to an ouptut directory, no trailing slash},target=/app/data inventory   
-  ```
 
 #### With a Virtual Environment
 
 You can also set up the application using `virtualenv` by doing the following:
 
 1. Create a virtual environment using `virtualenv`.
-   ```
-   virtualenv venv
-   source venv/bin/activate  # for Mac OS
-   ```
-1. Install the dependencies specified in `requirements.txt`.
-   ```
-   pip install -r requirements.txt
-   ```
-1. Run the application.
-   ```
-   python inventory.py
-   ```
-
-### Resources
+    ```
+    virtualenv venv
+    source venv/bin/activate  # for Mac OS
+    ```
+
+2. Install the dependencies specified in `requirements.txt`.
+    ```
+    pip install -r requirements.txt
+    ```
+
+3. Initialize the database using `create_db.py`.
+    ```
+    python create_db.py
+    ```
+
+4. Run the application.
+    ```
+    python inventory.py
+    ```
+
+
+#### OpenShift Deployment
+
+Deploying the application as a job using OpenShift and Jenkins involves several steps, which are beyond the scope of
+this README. However, a few details about how the job is configurd are provided below.
+
+* The `env.json` file described in the **Configuration** section above needs to be made available to 
+  running course-inventory containers via an OpenShift ConfigMap, a type of Resource. A volume containing the ConfigMap 
+  should be mapped to the `config/secrets` subdirectory. These details will be specified in a configuration file
+  (.yaml) defining the pod.
+
+* By default, the application will run with the assumption that the JSON configuration file will be named `env.json`. 
+  However, `inventory.py` will also check for the environment variable `ENV_PATH`. This variable can be set using the 
+  OpenShift pod configuration file. To use a different name for the JSON file, set `ENV_PATH` to a path beginning with 
+  `config/secrets/` and ending with the file name. With this set, the `env` block in the `.yaml` will look 
+  something like this:
+  ```
+  - env:
+      - name: ENV_FILE
+        value: /config/secrets/env_test.json
```

Comment:
env_test.json for the test server? Or should it be env.json

Reply:
It's an example of re-naming the `env.json` file, so I used something else here. Is that okay?